translation/file_loader.go

```diff
@@ -22,38 +18,25 @@ func NewFileLoader(paths []string) *FileLoader {
 	}
 }
 
-func (f *FileLoader) Load(folder string, locale string) (map[string]map[string]string, error) {
-	translations := make(map[string]map[string]string)
+func (f *FileLoader) Load(locale string, folder string) (map[string]map[string]any, error) {
+	translations := make(map[string]map[string]any)
 	for _, path := range f.paths {
-		var val map[string]string
-		fullPath := path
-		// Check if the folder is not "*", and if so, split it into subFolders
-		if folder != "*" {
-			subFolders := strings.Split(folder, ".")
-			for _, subFolder := range subFolders {
-				fullPath = filepath.Join(fullPath, subFolder)
-			}
-		}
-		fullPath = filepath.Join(fullPath, locale+".json")
+		var val map[string]any
+		fullPath := filepath.Join(path, locale, folder+".json")
```

Comment:
I think it is not that confusing, though. We can rename it to group. Control related to different types of file loading is under file_loader. So instead of adding file extension to it we will just send the name of file with folder.

Reply:
Not really ~ 
1. If we need to support other types files in the future, this method is not good, it's better to pass a file with suffix;
2. It's very strange if pass a file but the file doesn't have a suffix;

Reply:
Based on the Laravel code, `folder` renamed to `group` makes sense, and doesn't need to contain the suffix. Thanks for your check. 👍